README.md

```diff
@@ -29,3 +29,42 @@
 ## Homework Assignment -- due Friday April 12th
 - Complete [Labs](/labs)
 - Complete [Final project proposal](assignment)
+
+Final Project proposal
+Team Member: Sagari Datta, Dhruvi Kothari, Lufeng Lin
+
+Problem / Question:
+Traffic Jam Prediction in Louisville.
+To help the City of Louisville have a better knowledge of the city’s congestion, we have built a machine learning model to predict the hourly traffic jam across the City of Louisville. The application is to visualize and inform the prediction in an interactive way.
+
+The data:
+Waze traffic jam prediction dataset, which will contain the length of the jam, the coordinates, the date and the specific time of the jam. The prediction is for all 24 hours of the day for 7 days a week in 2018.
+
+
+Technologies used:
+jQuery, underscore, leaflet, chart.js
+
+Design spec:
+Targeted User:
+Transportation Planners from the City of Louisville
+
+Use Case:
+With the knowledge of future traffic congestion, planners and policy-makers are capable of doing many things. For example, in terms of avoiding traffic congestion, better site a place and schedule a time for an event.
+
+Anticipated difficulties:
+Backend dataset management for frontend application - We have selected the main highways and primary streets in Louisville using the OpenStreetMap data. We then created a total of 1,949 hexagonal grid cells (of 500 square meters each). We are predicting traffic delay for 7 days in a week for all 24 hours which means we have a prediction dataset with 327,432 rows (7 days x 24 hours x 1,949 fishnets). We are running the prediction model for all four directions separately and therefore will have four datasets with 327,432 rows.
+
+Display: We would like to include sliders for our 24 jam delay prediction when the user zooms into a particular grid cell. However, we will have to filter out the grid cell and select predictions by the hour and display it on the slider real time - we are concerned that loading time would be too long.  
```

Comment:
One option that would *easily* reduce rendering times would be to cut the data down to areas in which jams are predicted. I assume that's only a small % of the total road network. If my assumption is right, you might be able to sidestep the difficulty of rendering all 2000 cells at once